safe-global/safe-wallet-web
# fix: allow subscribing to owned but not added safes

## What it solves

Resolves #3288

## How this PR fixes it
- Do not show added but not owned safes in global notification settings.
- Show owned safes plus added & owned safes.
- Remove the notification banner completely.

## How to test it
1. Go to the settings->notifications -> global preferences
2. You should see the option to add notifications to only safes that are owned by the connected wallet.
---
1. Observe that the notification banner does not show up under any conditions.


## Checklist
